### ISSUE TITLE
Confirm password field in sign up page

### ISSUE THREAD

**Issue by shraddha761** [Counselllor/Counsellor-Web#217]
**Is your feature request related to a problem? Please describe.**
There should be confirm password field in sign up page
Assign this issue to me under GSSOC'23

**Comment by shraddha761** [comment]
@thestarsahil please assign this issue to me

**Comment by thestarsahil** [comment]
Elaborate this !

**Comment by shraddha761** [comment]
In the sign up page there should be confirm password field as user is registering for the first time. I want to add that field

**Comment by thestarsahil** [comment]
Yeah, I got it earlier, but I'm wondering how you will accomplish that. I'd like you to elaborate on the work you'll be doing. Could you please provide the starting point and the ending point?

**Comment by shraddha761** [comment]
I am gonna add confirm password field just below the password field and implement validation  whether password and confirm password are correct or not
